share/lib/python/scripts/nrniv-core

```diff
@@ -0,0 +1 @@
+_binwrapper.py
```

Comment:
I didn't create this right away because I wasn't sure what happens if coreneuron is not activated. Just check if there is dangling link to nrniv-core.